README.md

```diff
@@ -25,16 +25,14 @@ Parametric Connector Framework (PCF) follows a declarative paradigm and aims to
 # Getting Started
 
 
-```bash
+```sh
```

Comment:
@jackson-at-bentley why switch away from bash?

Reply:
`typedoc` didn't like it. I can't remember why anymore, but maybe `typedoc` doesn't use [`linguist`](https://github.com/github/linguist)? [Here's a list of supported languages](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml). I'll look into it.